tests/unit/resources/help.txt

```diff
@@ -29,7 +29,9 @@ Commands:
     server stop
     server uninstall
     server upgrade
+    slider install
```

Comment:
I thought we wanted this to be called server install ? Because this is not installing slider but setting up Presto on YARN using slider. AND 'slider slider_install' will be just 'slider install'.